front/src/reducers/osrdconf/types.ts

```diff
@@ -81,3 +82,8 @@ export type PathStep = PathItemLocation & {
     trackNumber: number;
   };
 };
+
+export type StdcmPathStep = PathStep & {
+  isVia: boolean;
```

Comment:
I think we can drop this field since it's defined below as well

Reply:
Seems like the field is still here, am I missing something?

Reply:
Oops, sorry, it's removed now